ddtrace/propagation/_database_monitoring.py

```diff
@@ -25,23 +25,24 @@
 log = get_logger(__name__)
 
 
-def _default_sql_injector(dbm_comment, sql_statement):
+def default_sql_injector(dbm_comment, sql_statement):
     # type: (str, str) -> str
```

Comment:
can we update the type hints here:

we need to have a `Union[str, bytes]` right? and same for the return `Union[str, bytes]`

(I don't know how to force the return type to be the same as the input type for `sql_statement`, since that is the actual ideal implementation)